client/webserver/site/src/html/markets.tmpl

```diff
@@ -263,13 +263,139 @@
+
+                    <div data-tmpl="quoteFeeReservesBox" class="d-flex justify-content-between align-items-center">
+                      <span>
+                        <span>[[[Fee reserves]]]</span>
+                        <img data-tmpl="quoteFeeLogo" class="micro-icon">
+                      </span>
+                      <span>
+                        <span data-tmpl="quoteFeeReserves"></span>
+                        <span data-tmpl="quoteFeeTicker"></span>
+                      </span>
+                    </div>
+
+                    <div data-tmpl="pendingDepositBox" class="d-flex justify-content-between align-items-center">
+                      <span>[[[Pending deposits]]]</span>
+                      <span data-tmpl="pendingDeposits"></span>
+                    </div>
+
+                    <div data-tmpl="pendingWithdrawalBox" class="d-flex justify-content-between align-items-center">
+                      <span>[[[Pending withdrawals]]]</span>
+                      <span data-tmpl="pendingWithdrawals"></span>
+                    </div>
+
+                    <div class="d-flex justify-content-between align-items-center">
+                      <span>[[[Settled matches]]]</span>
+                      <span data-tmpl="completedMatches"></span>
+                    </div>
+
+                    <div data-tmpl="tradedUSDBox" class="d-flex justify-content-between align-items-center">
+                      <span>[[[Traded]]]</span>
+                      <span>
+                        <span data-tmpl="tradedUSD"></span>
+                        <span class="fs14 grey">USD</span>
+                      </span>
+                    </div>
+
+                    <div class="d-flex justify-content-between align-items-center">
+                      <span>[[[Basis price]]]</span>
+                      <span data-tmpl="basisPrice"></span>
+                    </div>
+
+                    <div class="d-flex justify-content-between align-items-center">
+                      <span>[[[Fee gap]]]</span>
+                      <span>
+                        <span data-tmpl="feeGap"></span> (<span data-tmpl="feeGapPct"></span>%)
+                      </span>
+                    </div>
+
+                    <div data-tmpl="remoteGapBox" class="d-flex justify-content-between align-items-center">
+                      <span>[[[Remote gap]]]</span>
+                      <span>
+                        <span data-tmpl="remoteGap"></span> (<span data-tmpl="remoteGapPct"></span>%)
+                      </span>
+                    </div>
+
+                    <div data-tmpl="roundTripFeesBox" class="d-flex justify-content-between align-items-center">
+                      <span>[[[Round_trip fees]]]</span>
+                      <span>
+                        <span data-tmpl="roundTripFeesUSD"></span>
+                        <span class="fs14 grey">USD</span>
+                      </span>
+                    </div>
```

Comment:
I think a link to the `/mmlogs` page for this run would be good. These stats could be over there as well. I think the info about the market i.e. basis price, fee gap etc. would be more useful here, and then data about the run could be over there.